src/util/rename_symbol.cpp

```diff
@@ -171,6 +171,14 @@ bool rename_symbolt::rename(typet &dest) const
       result = false;
     }
 
+    const exprt &spec_frees =
+      static_cast<const exprt &>(dest.find(ID_C_spec_frees));
+    if(spec_frees.is_not_nil() && have_to_rename(spec_frees))
+    {
+      rename(static_cast<exprt &>(dest.add(ID_C_spec_frees)));
+      result = false;
```

Comment:
I think we don't have coverage here, because this will be use later in the back-end implementation. IS that accurate?